api/services/ccd-store-api/ccd-store.js

```diff
@@ -0,0 +1,53 @@
+const express = require('express');
+const config = require('../../../config');
+const generateRequest = require('../../lib/request');
+const mockRequest = require('../../lib/mockRequest');
+
+const url = config.services.ccd_data_api;
+
+function getHealth(options) {
+    return generateRequest('GET', `${url}/health`, options);
+}
+
+function getInfo(options) {
+    return generateRequest('GET', `${url}/info`, options);
+}
+
+function getOptions(req) {
+    return {
+        headers: {
+            Authorization: `Bearer ${req.auth.token}`,
+            ServiceAuthorization: req.headers.ServiceAuthorization
+        }
+    };
+}
+
+function getCCDCases(userId, jurisdictions, options) {
+    console.log(" inside getCCDCases");
+    const promiseArray = [];
+    if (process.env.JUI_ENV === 'mock') {
+        jurisdictions.forEach(jurisdiction => {
+            promiseArray.push(mockRequest('GET', `${config.services.ccd_data_api}/caseworkers/${userId}/jurisdictions/${jurisdiction.jur}/case-types/${jurisdiction.caseType}/cases?sortDirection=DESC${jurisdiction.filter}`, options))
```

Comment:
long term we should probally apply the request param from the otherside of the call but its fine for now